client/ayon_core/pipeline/editorial.py

```diff
@@ -224,38 +242,30 @@ def get_media_range_with_retimes(otio_clip, handle_start, handle_end):
     offset_in *= time_scalar
     offset_out *= time_scalar
 
-    # filip offset if reversed speed
+    # flip offset if reversed speed
     if time_scalar < 0:
-        _offset_in = offset_out
-        _offset_out = offset_in
-        offset_in = _offset_in
-        offset_out = _offset_out
+        offset_in, offset_out = offset_out, offset_in
 
     # scale handles
     handle_start *= abs(time_scalar)
     handle_end *= abs(time_scalar)
 
-    # filip handles if reversed speed
+    # flip handles if reversed speed
     if time_scalar < 0:
-        _handle_start = handle_end
-        _handle_end = handle_start
-        handle_start = _handle_start
-        handle_end = _handle_end
+        handle_start, handle_end = handle_end, handle_start
 
     source_in = source_range.start_time.value
 
-    media_in_trimmed = (
-        media_in + source_in + offset_in)
```

Comment:
If I understood correctly the logic here, I think adding `media_in` + `source_in` was wrong but still gave false-positive cause most media available range starts from 0.

Reply:
Media sources often start with metadata for the starting timecode, so they're not always beginning from `0`.